### DISCUSSION TITLE
Time picker

### DISCUSSION THREAD

**Discussion by yashikakhurana** [quasarframework/quasar#10395] [Ideas / Proposals] [answered]
Hey, I am just wondering does anyone know how to change time picker from clock style to dropdown style something similar to like this: 
![MicrosoftTeams-image (3)](https://user-images.githubusercontent.com/25848231/129581023-2c3837da-17bd-45c9-bdac-11f710d34a09.png)
Thanks in advance!

**Comment by ontwikkelfabriek** [comment, marked as answer]
@yashikakhurana 

You could look at this Quasar scroller plugin:
https://github.com/quasarframework/quasar-ui-qscroller

Live demos:
https://quasarframework.github.io/quasar-ui-qscroller/demo
https://quasarframework.github.io/quasar-ui-qscroller/examples/string

**Reply by yashikakhurana** [reply]
Hi, Thank you for sharing that, but my requirement is to get sliders for hours minutes and seconds, in q scroller it only gives the option for hours and minutes. Any idea how I could achieve that?

**Reply by ontwikkelfabriek** [reply]
@yashikakhurana 

> Any idea how I could achieve that?

You'll have to mod in seconds yourself. Most difficult part is already been done, the wheel UI part.
